rel-eng/tito.props

```diff
@@ -358,8 +358,6 @@ whitelist = rubygem-angular-rails-templates
   rubygem-ffi
   rubygem-fog-kubevirt
   rubygem-fog-proxmox
-  rubygem-foreman-tasks
```

Comment:
Inconsistent locale sorting is annoying. Perhaps we should rename things to be consistent.